### ISSUE TITLE
label:question - Form Builder?

### ISSUE THREAD

**Issue by elamotte** [icebob/vue-form-generator#327]
I want to create a form builder to be able to generate schema for VFG. Do you guys know anything like that? something I could start from?

**Comment by DelfsEngineering** [comment]
It is sounding like you want dynamic forms. I currently do this and it works great. ... Or are you looking for a tool that allows you to dynamically build forms with a UI that also allows for configuration (drag / drop etc) ?

**Comment by ChadTaljaardt** [comment]
I have made a custom form builder where you select the elements you want and it allows you to generate the schema, however this is a private project. Though, when developing i created a little prototype in my spare time that i can share. 

Give me a few hours and ill send a link to it.

**Comment by elamotte** [comment]
@DelfsEngineering yeah my form are already dynamics, and I store them in database. But I want to push it next step ahead ang give a UI tool to users to allow them modify/create their own form schemas. I started that actually for my project but it's very for my custom needs so not very sharable either. But sure @ChadTaljaardt if you have already something I would be curious to see it :)

**Comment by ChadTaljaardt** [comment]
@elamotte 

Here is a simple prototype, this was work i did in my own time. Its been heavily improved however i cannot share the upgrade. 

https://jsfiddle.net/en48m3py/4/

**Comment by elamotte** [comment]
@ChadTaljaardt thanks! 
This is a good starting point actually for now I will stay with my custom solution since it's fill my need but maybe if I have time I'll start a small repo with a component to generate a schema with basic UI.

The fun things in my actual version is that I even use VFG to generate the "form-edit" for each field, so vfg-ception! :)

**Comment by humblecoder** [comment]
@DelfsEngineering @ChadTaljaardt yeah, I'm also looking for something similar to [jQuery Form Builder](http://formbuilder.readthedocs.io/en/latest/).  A UI to construct these forms would be great!
